daemons/based/pacemaker-based.h

```diff
@@ -55,12 +55,11 @@ enum cib_op_attr {
 };
 
 typedef struct cib_operation_s {
-    const char *operation;
-    enum cib_op_attr flags;
+    const char *name;
+    uint32_t flags; //!< Group of <tt>enum cib_op_attr</tt> flags
```

Comment:
FYI the type correction is in the cib_get_operation() commit instead of the booleans commit, but no big deal

Reply:
moved this while fixing the xmlFreeDoc issue